src/segment-loader.js

```diff
@@ -646,13 +646,23 @@ export default class SegmentLoader extends videojs.EventTarget {
       return videojs.createTimeRanges();
     }
 
-    if (this.startingMedia_.hasAudio && this.startingMedia_.hasVideo) {
-      return this.sourceUpdater_.buffered();
-    } else if (this.startingMedia_.hasAudio) {
-      return this.sourceUpdater_.audioBuffered();
+    if (this.loaderType_ === 'main') {
```

Comment:
What do you think of doing something like this right above this if to prevent all the weird formatting: `const {hasAudio, hasVideo} = this.startingMedia_;`

Reply:
Good call. Updated.